### ISSUE TITLE
Footer - redesign

### ISSUE THREAD

**Issue by Dricmoy** [ratemystudyspot/website#18]
Support | RateMyStudySpot (logo)
Contact Us | About Us
these will be in the center

@2024 Ratemystudyspot .Privacy .Terms      English (CA) Discord Github Instagram  Linkedin
these will be similar to airbnb's bottom part

_Originally posted by @Dricmoy in https://github.com/ratemystudyspot/website/issues/16#issuecomment-2302739083_

**Comment by Dricmoy** [comment]
@LeoShangTang close the old issue for me ty 👍

**Comment by rayzhou4** [comment]
> Support | RateMyStudySpot (logo) Contact Us | About Us these will be in the center
> 
> @2024 Ratemystudyspot .Privacy .Terms English (CA) Discord Github Instagram Linkedin these will be similar to airbnb's bottom part
> 
> _Originally posted by @Dricmoy in [#16 (comment)](https://github.com/ratemystudyspot/website/pull/16#issuecomment-2302739083)_

This looks like a good design, try it out and let me know!

**Comment by Dricmoy** [comment]
Prototypes:
![image](https://github.com/user-attachments/assets/4d7c55f6-e9f8-43a9-be1f-152a996b2eb4)

mobile is basically similar, but this is all the footer needs to have right, we haven't decided on colours or anything? @rayzhou4

**Comment by rayzhou4** [comment]
> Prototypes:
> 
> ![image](https://github.com/user-attachments/assets/4d7c55f6-e9f8-43a9-be1f-152a996b2eb4)
> 
> 
> 
> mobile is basically similar, but this is all the footer needs to have right, we haven't decided on colours or anything? @rayzhou4 

Yeah, looks good to me! Maybe try out a few different styles with where the logo should be though, or maybe just get rid of it, cause it looks a bit out of place right now

**Comment by Dricmoy** [comment]
yea it looks really out of place, no matter where I put it

**Comment by LeoShangTang** [comment]
Maybe do something like github??

![image](https://github.com/user-attachments/assets/98b1820e-7a63-41f4-8109-efc940b9b842)

**Comment by rayzhou4** [comment]
I like the github suggestion, if theres any way to implement it like theirs, that’d be awesome

**Comment by LeoShangTang** [comment]
Making the logo a smaller size similar to the github one should be good enough

I did not mean to close the issue LOL
